app/scripts/services/metrics.js

```diff
@@ -229,6 +244,84 @@ angular.module("openshiftConsole")
       });
     };
 
+    var getMetricInfo = function(metricID) {
+      // Example descriptor:
+      //   ruby-helloworld/6f7881e1-52c4-11e6-a5dc-080027893417/memory/usage
+      var segments = metricID.split('/');
+      return {
+        podUID: segments[1],
+        descriptor: segments[2] + '/' + segments[3]
```

Comment:
I don't think they will change. They're documented at least.

https://github.com/kubernetes/heapster/blob/1b2de45a3e07c85e016788584b4ea98d7cdaada0/docs/storage-schema.md#hawkular

We're already relying on them in our URLs.

https://github.com/spadgett/origin-web-console/blob/d5715f3ac2cd7aa30e75a6c80309fea113deefe3/app/scripts/services/metrics.js#L5-L6